README.md

```diff
@@ -311,6 +329,105 @@ public function broadcastAs($event)
 
 Remember that if you don't use `broadcastAs()` at all, Laravel will default to use the event class name, and here for an Order model for example you could see `OrderCreated` as the `Subject`.
 
+## Subscribing / Listening
+
+### Configuration
+
+Once the package is installed and similar to what you would do for a standard Laravel SQS queue, you will need to add the following connection and configure your credentials in the `config/queue.php` configuration file:
+
+```php
+'connections' => [
+    // ...
+    'sqs-sns' => [
+        'driver' => 'sqs-sns',
+        'key' => env('AWS_ACCESS_KEY_ID'),
+        'secret' => env('AWS_SECRET_ACCESS_KEY'),
+        'prefix' => env('SQS_SNS_PREFIX', 'https://sqs.us-east-1.amazonaws.com/your-account-id'),
+        'queue' => env('SQS_SNS_QUEUE', 'pub-sub'),
+        'suffix' => env('SQS_SNS_SUFFIX'),
+        'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
+    ],
+    // ...
+],
+```
+
+Once your queue is configured properly, you will need to be able to define which listeners you would like to use for which kind of incoming events. In order to do so, you'll need to create Laravel Listeners and associate the events through a Service Provider the package can create for you.
```

Comment:
We are inconsistent with whether listeners should have a capital or not throughout this file.